### PR TITLE
feat(render): Add forced init option on lazy rendering

### PR DESCRIPTION
**Pull request by netil**
## Issue

#3106

## Details

Make to initialize when chart element isn't visible, but render.lazy=false is set

### CODE REVIEW

File: src/Chart/Chart.ts [hunk 1]
```diff
@@ -137,6 +142,12 @@ export default class Chart {
 
 		$$.beforeInit();
 		$$.init();
+
+		// if ($$.config.render.lazy !== false && hasStyle($$.$el.chart, {"display": "none", "visibility": "hidden"})) {
+		// 	hook = () => {
+		// 		logError(`The call of APIs won't work. Please, make sure if chart element is %cvisible.`);
+		// 	};
+		// }
```
Comment (stof) [inline src/Chart/Chart.ts:150]: shouldn't this has been removed ?

File: test/internals/bb-spec.ts [hunk 2]
```diff
@@ -518,7 +518,7 @@ describe("Interface & initialization", () => {
 		}));
 
 		it("check lazy rendering on callbacks", () => new Promise(done => {
-			const el: any = document.body.querySelector("#chart");
+			const el = <HTMLDivElement>document.body.querySelector("#chart");
```
Comment (stof) [inline test/internals/bb-spec.ts:521]: the modern typescript way for that is to use the `document.body.querySelector("#chart") as HTMLDivElement` instead.

File: src/Chart/Chart.ts (hunk 1 above)
Comment (netil) [reply src/Chart/Chart.ts:150]: Left for future consideration as comment.

File: test/internals/bb-spec.ts (hunk 2 above)
Comment (netil) [reply test/internals/bb-spec.ts:521]: Thanks for the suggestion.

**Pull request merged**